### PR TITLE
Added meta tags

### PR DESCRIPTION
**Pull request by murali-krishna-sv**

### CODE REVIEW

**Comment by benoitf** [comment]
PR looks nice but a lot of content is reformatted.
I guess the PR should only introduce changes, not formatter changes else we don't see clearly what is being done

File: src/lib/stores.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { writable } from 'svelte/store';
+
+export const pageTitle = writable('Home');
+export const pageDescription = writable('Run your favorite IDE on Kubernetes');
+export const pageUrl = writable('');
```
Comment (ibuziuk) [inline src/lib/stores.ts:5]: no New Line

**Review by ibuziuk — COMMENTED** [review]
@murali-krishna-sv thank you for the contribution 👍 
could you please address the review comments / formatting

**Comment by murali-krishna-sv** [comment]
Creating a new PR with the changes.
https://github.com/eclipse-che/che-website/pull/75

**Pull request closed without merging**